tests/unit_tests/fn/test_fn_backend_gpu.cu

```diff
@@ -12,6 +12,7 @@
 #include <gtest/gtest.h>
```

Comment:
I would prefer having this test as a regression test, running for all available backends. You could even just put a new variant of the copy stencil test into `tests/regression/fn/fn_copy.cpp` I guess. There are already versions with and without domain offsets, so the one with zero domain size would actually fit well.

Reply:
…or add a separate one as you also check the scan. I don’t care much, but enabling it for all backends makes sense for sure.

Reply:
I made a separate regression test (and removed the unit test).